streamerbot/3.api/1.sub-actions/core/commands/Get-Commands.md

```diff
@@ -88,4 +88,4 @@ public class CPHInline
 ---
 
 - [<i class="mdi mdi-chevron-left"></i> **Commands Sub-Actions Reference *Go Back***](/Sub-Actions/Commands)
```

Comment:
This whole section can be removed as well.  Artifacts left over from the old wiki system.
[<i class="mdi mdi-chevron-left"></i> **Commands Sub-Actions Reference *Go Back***](/Sub-Actions/Commands)
{.btn-grid .my-5}